August Beta Bugfixes

ilkober opened the pull request:
Accept late

ilkober committed:
some code cleanup for LIE + one outdated idea check removed

ilkober committed:
Same here - just fixing some errors

ilkober committed:
Update liechtenstein.txt

ilkober committed:
Moving decision icons to proper folder, correct naming of ISR event files

ilkober committed:
Portrait fixes (ssee desc)

Fixes
Fixed Genovevo Perez Damera portrait.
Fixed Luxembourgian advisor portrait not displaying correctly

ilkober committed:
Misc clean ups

hammmurabi committed:
Update KOR.txt

hammmurabi committed:
some fixes

BiccyThiccy committed:
Porting over most of the vanilla FAS-SPA changes

One of the recent War Effort's made a bunch of changes to Spain's fascist path.

The only thing from there I haven't ported over is the reworked Retake the America's sub-branch. Given every change besides that is a very simple copy-paste, I figure it's worth adding to the patch.

Spain will likely get more love from me down the line.

ilkober committed:
Clean-up

ilkober committed:
a

marijn211 committed:
Fix broken Albania tree after YUG overthrow

TheBrit79 committed:
Minor things

TheBrit79 committed:
Merge branch 'august-beta-bugfixes' of https://github.com/deliciousmods/1956_beta into august-beta-bugfixes

marijn211 reviewed (dismissed):
Should be checked whether the lack of OOBs is an issue

marijn211 commented:
@TheBrit7 one of the adjecencies has an error

marijn211 committed:
Country name for Sint Maarten

BiccyThiccy committed:
Belgium partial cleanup

- Will_lead_to_war added to fit vanilla standards
- ai_will_do's now use base = _ correctly.

Even besides it being an old tree, Belgium needs a decent amount of work (character conversion especially). This will solve a lot of gameplay annoyances with surprise-wars though.

ilkober committed:
Resolving  some code errors

ilkober committed:
BftB AI + event dispatcher

BftB: Turkish fundamentalists demands for the  closure of factories should appear as intended.
BftB: Bulgarian allies will be more likely to get annexed if tension is very high

ilkober committed:
No new error log popups

hammmurabi committed:
Merge branch 'Beta-Edits' into august-beta-bugfixes

hammmurabi reviewed (approved):
falkland strait crossing has an error but seems to be working as intended, and the removal of the OOBs doesnt seem to have any issues either, merging.
